src/configuration/services/mysql.md

```diff
@@ -199,6 +200,17 @@ ALTER TABLE table_name CONVERT TO CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_
 
 Consult the [MySQL documentation](https://dev.mysql.com/doc/refman/8.0/en/charset-mysql.html) for further details.
 
+## Storage Engine
+
+We recommend using the InnoDB storage engine wherever possible. MyISAM is only properly supported in Grid environments, in dedicated cluster environments there is no replication of MyISAM tables.
+
+If MyISAM tables are inadventently created or imported in a dedicated environment they can be easily converted to use the InnoDB storage engine using the following proceedure:
+
+```sql
+RENAME TABLE <existing> <old>;
+INSERT INTO <existing> SELECT * from <old>;
```

Comment:
Wouldn't a rename include data?  This seems... odd.  I think the first is supposed to be a create statement or something, no?

Reply:
It's a rename. Essentially moving it out of the way and then recreating it via the INSERT INTO so that it gets recreated with the correct, default storage engine and proper replication.